include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h

```diff
@@ -54,6 +54,13 @@ struct OpBinder {
     return success();
   }
 
+  ParseResult tensorOperandsList( llvm::SmallVector<Value, 4> &values) {
+    for (int i = 0; i < op->getNumOperands(); i++) {
+      values.push_back(op->getOperand(i));
+    }
+    return success();
+  }
+
```

Comment:
Something similar is done here: https://github.com/llvm/torch-mlir/pull/2601/files#diff-473c414724482953c052f65a2f0eb1f8387f62ba9793ac04474437c0e9dd8c2dR57-R59

Reply:
Ok